pkg/machines/libvirt.es6

```diff
@@ -170,29 +175,47 @@ LIBVIRT_PROVIDER = {
         };
     },
 
+    INIT_DATA_RETRIEVAL () {
+        logDebug(`${this.name}.INIT_DATA_RETRIEVAL():`);
+        return dispatch => {
+            dispatch(getOsInfoList());
+            return cockpit.script(getLibvirtServiceNameScript, null, { err: "message", environ: ['LC_ALL=en_US.UTF-8'] })
+                .then(serviceName => {
+                    const match = serviceName.match(/(.+)\n$/);
+                    const name = getRegexGroup(match, 1, 'libvirtd.service'); // fallback to libvirtd to be able to show link in libvirtSlate
```

Comment:
I don't think this should fall back in this way. If libvirtd isn't installed (i. e. the above command does not have any result), then this should be treated properly and not ignored. If libvirt isn't installed, then getting its state and VMs is doomed to fail anyway.

Reply:
Ok I will move this handling to libvirtSlate and leave serviceName empty in the store